modules/test/playwright/tests/object-web/modelBuilder.spec.ts

```diff
@@ -86,3 +86,85 @@ test('can create relationship by dragging node handles', async ({
 
 	await apiHelpers.objectAdmin.deleteObjectFolder(objectFolder.id);
 });
+
+test('can delete object relationship from different folders', async ({
+	apiHelpers,
+	modelBuilderPage,
+	objectDefinitionsPage,
+}) => {
+	await apiHelpers.featureFlag.updateFeatureFlag('LPS-148856', true);
```

Comment:
i think the FF for the model builder was already removed right? can we delete this line?